com.github.vikdevelop.googlemaps.yml

```diff
@@ -0,0 +1,46 @@
+app-id: com.github.vikdevelop.googlemaps
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions: 
+  - org.freedesktop.Sdk.Extension.node16
+command: run.sh
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=wayland
+  - --socket=fallback-x11
+modules:
+  - name: googlemaps
+    buildsystem: simple
+    build-options:
+       append-path: /usr/lib/sdk/node14/bin
```

Comment:
seems like a mismatch between the node16 extension and node14 here? (I haven't looked at this in detail, it may not be)